src/components/layout/location-context.tsx

```diff
@@ -9,7 +9,7 @@ type LocationProviderProps = {
     toggleModal?: (event: React.MouseEvent<HTMLElement>) => void
 }
 
-export const LocationContext = React.createContext(null) // undefined => true
+export const LocationContext = React.createContext(true) // undefined => true
```

Comment:
```suggestion
export const LocationContext = React.createContext(true)
```
@mitra-deriv The comment isn’t valid anymore, We can remove it to avoid confusion in the future.

Reply:
Hi @farzin-fs , Sure will do. It was there only before as well thats why I kept. 🌹